crates/astria-sequencer-types/src/sequencer_block_data.rs

```diff
@@ -387,4 +461,23 @@ mod test {
         let actual = SequencerBlockData::from_bytes(&bytes).unwrap();
         assert_eq!(data, actual);
     }
+
+    #[test]
+    fn test_calculate_last_commit_hash() {
```

Comment:
this seems useful, in a scenario like testing `last_commit_hash` we can use one of the default proxy apps (https://github.com/cometbft/cometbft/tree/6a7dfb40d0302de5cdbca51cc4daede74b6d44c2/abci/example/kvstore) since we're just testing against cometbft, not our sequencer app. however i'd like to still have a unit test for this function that doesn't requiring external binaries/spinning up additional processes, so i think this test is useful still, maybe i can reframe is as "test vectors from a specific cometbft version" and add steps on how those vectors were generated?

Reply:
yeah, definitely doesn't hurt to have an integration test!